src/libNetwork/RumorManager.cpp

```diff
@@ -134,6 +135,7 @@ bool RumorManager::Initialize(const std::vector<Peer>& peers,
   m_peerIdSet.clear();
   m_selfPeer = myself;
   m_tmpRumorHashSet.clear();
+  m_rumorHashRawMsgBimap.clear();
```

Comment:
The default constructor should initialize empty bitmap already, why here need clean again?

Reply:
the initialize function is called every ds epoch.

Reply:
Ok, i see, i thought the clear is in constructor function.